Problems/3009/3009_ans_BNR.py

```diff
@@ -0,0 +1,17 @@
+#https://www.acmicpc.net/problem/3009
+
+x = []
+y = []
+for i in range(3):
+    A, B = list(map(int, input().split()))
```

Comment:
```suggestion
    A, B = map(int, input().split())
```
map의 인자를 각각 배정 할 경우, list 변환은 필요 없음.